website/static/js/comment.js

```diff
@@ -298,7 +298,12 @@ var CommentModel = function(data, $parent, $root) {
             'gravatarUrl': userData.links.profile_image
         };
     } else {
-        self.author = self.$root.author;
+        self.author = {
+            'id': null,
+            'urls': {'profile': ''},
+            'fullname': 'Deactivated user',
```

Comment:
Ah, yes. Seemingly because `api/base/utils.py:get_object_or_error` raises an error if a disabled user is requested.